packages/vsce/src/commands/disableResourceCommand.ts

```diff
@@ -10,18 +10,18 @@
  */
 
 import { IBundle, IJVMEndpoint, IJVMServer, ILibrary, ILocalFile, IProgram, IResource, ITransaction } from "@zowe/cics-for-zowe-explorer-api";
-import { commands, TreeView, window } from "vscode";
+import * as vscode from "vscode";
+import { TreeView, commands, window } from "vscode";
```

Comment:
Same here about the import

```suggestion
import { TreeView, commands, window, l10n } from "vscode";
```

Reply:
Done